v2.4/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -34,10 +34,10 @@ And then enable isolation on the Namespace using a [default policy](https://kube
 ```
 kubectl create -f - <<EOF
 kind: NetworkPolicy
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1
```

Comment:
I don't think we should make this change yet - this requires users to be on Kubernetes v1.7 - I think our guides should work on both v1.6 and v1.7, at least until v1.8 comes out.  Can you revert?